src/components/common/LocationMonitor.js

```diff
@@ -115,12 +116,11 @@ class LocationMonitor extends React.Component {
 }
 
 LocationMonitor.propTypes = {
-  getArea: PropTypes.func,
-  setLocation: PropTypes.func,
-  getNearbyPlaces: PropTypes.func,
-  coordinate: PropTypes.object,
-  timestamp: PropTypes.number,
-  provider: PropTypes.string,
+  getArea: propTypes.func.isRequired,
```

Comment:
Use either prop-types or flow but not both in same project. May be next commit replace prop-types definition with flow definition